src/mumble/AudioOutputSpeech.cpp

```diff
@@ -221,7 +222,7 @@ bool AudioOutputSpeech::prepareSampleBuffer(unsigned int frameCount) {
 
 	while (iBufferFilled < sampleCount) {
 		int decodedSamples = iFrameSize;
-		resizeBuffer(iBufferFilled + iOutputSize);
+		resizeBuffer(iBufferFilled + iOutputSize); // TODO: problematic line.
```

Comment:
If this is indeed the problematic line, I'd suggest removing it entirely. Instead I think we should probably just log an error in this case (probably crash in debug mode) and then discarding everything that doesn't fit in the current buffer.

Reply:
While... It works on most of the platform, and it has been here for many years. This is the first time it crashes the mumble. I'm not really sure about the mechanism of this crashing, and I think I'd wait for the result from @Reikion. Currently, I assume it is only problematic on some platforms using PulseAudio, though I have no idea what exactly those platforms are. I think as long as this is not a problem in other OS, I can find a way to avoid executing this line if the backend is PulseAudio.

Reply:
I consider that this is kinda similar to using undefined behavior in c++. It might work (for years) but if you're unlucky you'll find the situation in which you'll get the crash.

Thus **if possible** I think it'd be best to remove this line :thinking: